neuro-sdk/src/neuro_sdk/_buckets.py

```diff
@@ -228,13 +233,15 @@ async def create(
         self,
         name: Optional[str] = None,
         cluster_name: Optional[str] = None,
-        org_name: Optional[str] = None,
+        org_name: Union[Optional[str], OrgNameSentinel] = ORG_NAME_SENTINEL,
     ) -> Bucket:
         url = self._get_buckets_url(cluster_name)
         auth = await self._config._api_auth()
         data = {
             "name": name,
-            "org_name": org_name or self._config.org_name,
+            "org_name": org_name
+            if not isinstance(org_name, OrgNameSentinel)
```

Comment:
Or maybe just `if org_name is not ORG_NAME_SENTINEL`?

Reply:
In the current code, mypy cannot deduce that `OrgNameSentinel` is a singleton, so it can only narrow the type after `isinstance` check. Maybe `Literal` will can solve this problem.